src/js/reducers/projectValidationReducer.js

```diff
@@ -58,6 +59,11 @@ const projectValidationReducer = (state = initialState, action) => {
       ...state,
       onlyShowProjectInformationScreen: action.value,
     };
+  case types.SHOW_ONLY_PROJECT_INFORMATION_SCREEN_CALLBACK:
```

Comment:
Should we find a better name for this? I find it very confusing and cant figure out what it actually means